### PR TITLE
🎨 Improve Header, Nav and Mobile menu

### PR DESCRIPTION
**Pull request by peduarte**
Small improvement to the shared styles of Header, Nav and Mobile menu.

### CODE REVIEW

File: _sass/utilities/grid.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-	padding-left: $gutter * 1.5;
-	padding-right: $gutter * 1.5;
 	margin-left: auto;
 	margin-right: auto;
+	padding-left: $gutter * 1.5;
+	padding-right: $gutter * 1.5;
 
 	@include from("lg") {
-		max-width: 1024px;
+		max-width: map-get($breakpoints, "lg");
```
Comment (artisologic) [inline _sass/utilities/grid.scss]: This won't work because they are unitless.

Comment (peduarte) [inline _sass/utilities/grid.scss]: weird, it worked here. let me double check

Comment (peduarte) [inline _sass/utilities/grid.scss]: yep it was broken, fixed!

File: _sass/utilities/grid.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	padding-right: $gutter * 1.5;
 
 	@include from("lg") {
-		max-width: map-get($breakpoints, "lg");
+		max-width: unquote(map-get($breakpoints, "lg") + "px");
```
Comment (artisologic) [inline _sass/utilities/grid.scss]: Can we not do this instead:
`max-width: map-get($breakpoints, "lg") * 1px;`

Comment (peduarte) [inline _sass/utilities/grid.scss]: Yeah we _could_ but the other way round feels more concise to me. Don't you find it slightly hacky to multiple by 1px?

Comment (artisologic) [inline _sass/utilities/grid.scss]: Haha, I feel completely the opposite.
To have to use `unquote` feels hacky to me.
I believe * by the base unit is the recommended way to go from unitless to with unit.

**Pull request merged**